app/views/cats/index.html.erb

```diff
@@ -15,10 +15,10 @@
           <%= I18n.t 'cats.abandonedDate'%> <%= cat.abandoned_date %><br/>
         <% end %>
         <%= I18n.t 'cats.document'%> <%= cat.document %><br/>
-        <% if !current_user.nil? && current_user.role == 'volunteer' %>
-             <%= link_to "Edit", edit_cat_path(cat.id) %>
-             <%= link_to "Destroy", cat_path(cat.id), method: :delete %>
-
+        <% if !current_user.nil? && (current_user.role == 'volunteer' || current_user.role == 'admin') %>
```

Comment:
I think we can try use Pundit conditions here instead of writing the whole code. Example:

<% if policy(@cat).show? %>
  blah blah blah
<% end %>